com.github.TheThirdOne.rars.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.TheThirdOne.rars</id>
```

Comment:
it should be `io.github`

Reply:
RARS does not have any GitHub pages associated with it - should I still change it to presumably `io.github.TheThirdOne.rars`?

Reply:
Yes. `com.github` would be for apps submitted and maintained by GitHub themselves. `io.github` is for user-created content.